util/buildbot/buildwin32.sh

```diff
@@ -71,6 +71,8 @@ cmake_args=(
 	-DENABLE_GETTEXT=1
 	-DENABLE_LEVELDB=1
 )
+# Use ccache if it is available
+command -v ccache >/dev/null && cmake_args+=(-DCMAKE_{C,CCX}_COMPILER_LAUNCHER=ccache)
```

Comment:
I made a typo that causes ccache to never work...
```suggestion
command -v ccache >/dev/null && cmake_args+=(-DCMAKE_{C,CXX}_COMPILER_LAUNCHER=ccache)
```